frontend/appflowy_flutter/lib/shared/icon_emoji_picker/icon_picker.dart

```diff
@@ -250,43 +272,49 @@ class _IconPickerState extends State<IconPicker> {
               color: context.pickerTextColor,
             ),
             const VSpace(4.0),
-            Wrap(
-              children: iconGroup.icons.map(
-                (icon) {
-                  return widget.enableBackgroundColorSelection
-                      ? _Icon(
-                          icon: icon,
-                          mutex: mutex,
-                          onSelectedColor: (context, color) {
-                            widget.onSelectedIcon(
-                              IconsData(
-                                iconGroup.name,
-                                icon.content,
-                                icon.name,
-                                color,
-                              ),
-                            );
-                            PopoverContainer.of(context).close();
-                          },
-                        )
-                      : _IconNoBackground(
-                          icon: icon,
-                          onSelectedIcon: () {
-                            widget.onSelectedIcon(
-                              IconsData(
-                                iconGroup.name,
-                                icon.content,
-                                icon.name,
-                                null,
-                              ),
-                            );
-                          },
-                        );
-                },
-              ).toList(),
+            GridView.builder(
+              gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
+                crossAxisCount: widget.iconPerLine,
+              ),
+              itemCount: iconGroup.icons.length,
+              shrinkWrap: true,
```

Comment:
Would using `shrinkWrap` here cause the first-time loading to be slower?

Reply:
> Would using `shrinkWrap` here cause the first-time loading to be slower?

Using `shrinkWrap` is to allow the `GridView` to be properly nested within a ListView, and using `GridView` is to control the number of items per row. And the children of the GridView here won't be too many, so there is no need to worry about performance issues